src/sent2vec.pyx

```diff
@@ -96,3 +100,7 @@ cdef class Sent2vecModel:
 
     def embed_sentence(self, sentence, num_threads=1):
         return self.embed_sentences([sentence], num_threads)
+
+    @staticmethod
+    def release_shared_mem():
```

Comment:
A comment in the readme on how to release the shared memory would be nice. How is the cmd not taking into account the name of the model file?